robber/matchers/__init__.py

```diff
@@ -5,12 +5,12 @@
     'called_once',
     'called_once_with',
     'called_with',
-    'contain'
+    'contain',
     'equal',
-    'ever_called'
+    'ever_called',
```

Comment:
so I think that it's should be:
```
expect(callable).to.have.any_call(...)
```
should work? Would it be easier (since it's similar to the api of mock) to reason?